testnet/stacks-node/src/config.rs

```diff
@@ -59,6 +59,36 @@ pub struct LegacyMstxConfigFile {
 mod tests {
     use super::*;
 
+    #[test]
+    fn check_invalid_config_files() {
+        ConfigFile::from_path("some_path").expect_err("path does not exist");
+        ConfigFile::from_str("//[node]").expect_err("invalid toml comment");
```

Comment:
Stylistically, we generally use an explicit `assert!` for these kinds of checks:

```rust
assert!(ConfigFile::from_str("//[node]").is_err());
```